servicemesh-install/topics/appendix-threescale-adapter.adoc

```diff
@@ -69,8 +95,32 @@ The adapter embeds a tool that allows generation of the `handler`, `instance`, a
 |File to save produced manifests to
 |No
 |Standard output
+
+|`-v`
+|Outputs the CLI version and exits
+|No
+|
 |===
 
+=== Generate templates from URL examples
+This example generates the templates from the URL, service, and personal token:
+
+----
+3scale-gen-config --url="https://<organization>-admin.3scale.net:443" --service="123456789" --token="[redacted]"
+----
+
+This example generates the templates from the URL, service, and personal token but with a unique ID, that otherwise would have been derived from the URL:
+
+----
+3scale-gen-config --url="https://<organization>-admin.3scale.net" --uid="my-unique-id" --service="123456789" --token="[redacted]"
+----
+
+The previous example might fail to pass validation if the given unique ID did contains special characters. You can use the fixup mode to correct that:
```

Comment:
```suggestion
The previous example might fail to pass validation if the given unique ID contains special characters. You can use the fixup option to attempt to correct validation errors:
```

Reply:
I had to add this change manually locally. I'll push it up here on the next PR update.